src/styles/components/_info.sass

```diff
@@ -0,0 +1,393 @@
+			position: relative
+			width: 84px
+			right: 85px
+			top: 12px
+			transform: rotate(1deg)
+
+	&.pos-export
+		top: 241px
+		left: 150px
+		&:before
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			width: 85px
+			right: 86px
+			top: 10px
+			transform: rotate(5deg)
+
+	&.pos-magnet
+		top: 301px
+		left: 125px
+		&:before
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			width: 58px
+			right: 59px
+			top: 10px
+			transform: rotate(16deg)
+
+	&.pos-minimap
+		top: 355px
+		left: 115px
+		&:before
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			width: 56px
+			right: 54px
+			top: 2px
+			transform: rotate(30deg)
+
+	&.pos-profile
+		right: 266px
+		top: 72px
+		position: absolute
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -85px
+			width: 84px
+			top: -11px
+			transform: rotate(10deg)
+
+	&.pos-localization
+		top: 128px
+		position: absolute
+		right: 251px
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -70px;
+			width: 69px;
+			top: -7px;
+			transform: rotate(4deg);
+
+	&.pos-feedback
+		top: 187px
+		position: absolute
+		right: 242px
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -61px
+			width: 59px
+			top: -23px
+			transform: rotate(-7deg)
+
+	&.pos-logout
+		position: absolute
+		top: 249px
+		right: 267px
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -85px
+			width: 86px
+			top: -34px
+			transform: rotate(-9deg)
+
+	&.pos-boardname
+		position: 	absolute
+		top: 		234px
+		right: 		58%
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -85px
+			width: 86px
+			top: -7px
+			transform: rotate(13deg)
+
+	&.pos-boardpreview
+		position: 	absolute
+		top: 		330px
+		right: 		58%
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -170px
+			width: 170px
+			top: -7px
+			transform: rotate(10deg)
+
+	&.pos-boardbg
+		position: 	absolute
+		top: 		448px
+		right:  	58%
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -88px
+			width: 89px
+			top: -7px
+			transform: rotate(13deg)
+
+	&.pos-boardmeasures
+		position: 	absolute
+		top: 		520px
+		right:  	58%
+		&:after
+			border-top: 2px solid $affair
+			content: ' '
+			display: block
+			position: relative
+			float: right
+			right: -88px
+			width: 89px
+			top: -7px
+			transform: rotate(13deg)
+
+	&.pos-format //share & export bubles
```

Comment:
Don't comment on the same row as the actual code, keep comments on separate lines.